### PR TITLE
[Nature.com] Remove wildcard and update subdomains

### PR DESCRIPTION
**Pull request by Bisaloo**
It's a though one. I may be missing some rules but it shouldn't break anything (which matters more IMO).

Please also note that real https coverage for this domain is quite new (the cert is valid from 04/13 and I don't remember seeing it in https before today) so it might still be in progress.

### CODE REVIEW

**Comment by Bisaloo** [comment]
For some reason, travis times out on http://data.nature.com.

It's quite strange because it worked for a6bad68 but after just changing the ruleset name in 877c629, it stopped working.

File: src/chrome/content/rules/Nature.com.xml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		press.nature.com
+		proceedings.nature.com
+		protocols.nature.com
+		reports.nature.com
+		search.nature.com
+		split.nature.com
+		structbio.nature.com
+		support.nature.com
+		xml.nature.com
+		xps.nature.com
```
Comment (J0WI) [inline src/chrome/content/rules/Nature.com.xml]: my.nature.com

Comment (Bisaloo) [inline src/chrome/content/rules/Nature.com.xml]: Given the message over `http`:

> my.nature.com has been discontinued

> Following a beta pilot, my.nature.com discontinued service on April 14th, 2014.

> We apologize for any inconvenience caused.

is it really necessary to include it?

Comment (J0WI) [inline src/chrome/content/rules/Nature.com.xml]: IMHO: Since this host is still up there is no excuse to not support https.

Comment (Bisaloo) [inline src/chrome/content/rules/Nature.com.xml]: Looks like they noticed something was up and the subdomain is now unreachable even over `http`.

**Pull request merged**